data/intro missions.txt

```diff
@@ -451,3 +451,143 @@ conversation "end of intro missions"
+				complete
+		on complete
+		log `Met with James on <origin>, and filled him in on recent events with the war and the Pug.`
+
+
+
+mission "Intro: Follow Up - Hai After Main Plot"
+	name "Bring James to <planet>"
+	description "Take James to <destination> to meet the Hai."
+	blocked `You consider paying a visit to James to tell him about the Hai, but decide to wait until you have a free seat available on your ship. Just in case.`
+	passengers 1
+	landing
+	source "Hestia"
+	destination "Allhome"
+	to offer
+		or
+			has "Intro [3 Freighter]: done"
+			has "Intro [3 Transport]: done"
+			has "Intro [3 Interceptor]: done"
+		has "main plot completed"
+		has "First Contact: Hai: offered"
+	to fail
+		has "Intro: Follow Up - Hai Before Main Plot: offered"
+	on offer
+		conversation
+			`It has been a while since you last saw James. Do you want to check in and see how he is doing?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+			branch revisit
+				has "Intro: Follow Up - After Main Plot: done"
+			`	James is happy to come out to the spaceport and meet with you. You spend some time talking about recent events, and fill him in on some of the details of the recent war and the role you ended up playing in it. You talk a bit about the Pug and, after a time, the conversation comes around to the Hai. James is not completely surprised as your story confirms rumors that he has already heard.`
+				goto ride
+			label revisit
+			`	When you contact James to tell him that you are on visiting <origin>, he is happy to come out to the starport and meet you. The two of you talk a bit about what you have been up to since the war and, before long, the conversation comes around to the Hai. James is not completely surprised as your story confirms rumors that he has already heard.`
+			label ride
+			choice
+				`	"Why don't you come with me and see them for yourself?  There are a lot of retired spacers on <planet>."`
+				`	"It has been good catching up with you, James."`
+					goto end
+			`	James agrees. When you ask if he needs to collect any of his belongings before the trip he declines, but buys a simple traveling hygiene kit from one of the spaceport shops.`
+				accept
+			label end
+			`	You finish your conversation and go on your way`
+				decline
+	on decline
+		log `Paid a visit to James on <origin> to catch him up on recent events and to tell him about the Hai.`
+	on complete
+		conversation
+			`James is impressed by the friendly aliens and humans living side by side on <planet>. While you are introducing him to one of the Hai, an old spacer recognizes James and offers to make further introductions around the community on <planet>. You make your goodbyes once again, and prepare to continue on your way.`
+		log `Paid a visit to James on <origin> to catch him up on recent events and to tell him about the Hai. Afterwards, I dropped him off on <planet>.`
```

Comment:
The conversation should be the last thing under`on complete`, so move this log entry above the `conversation`.

Reply:
done

Reply:
It's a style thing.